src/components/SlateEditor/PlainTextEditor.tsx

```diff
@@ -73,7 +73,7 @@ const PlainTextEditor = forwardRef<HTMLTextAreaElement, Props>(
     const { status, setStatus } = useFormikContext<ArticleFormType>();
 
     useEffect(() => {
-      if (status?.status === "revertVersion") {
+      if (status?.status === "revertVersion" || status?.status === "acceptGenerated") {
```

Comment:
Dette er en grei løsning inntil videre, men synes vi burde gå for en mer solid løsning etterhvert